advocacy_docs/partner_docs/SIBVisionsGuide/02-PartnerInformation.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: 'Partner Information'
+description: 'Details for SIB Visions and Vision X'
+
+---
+
+| &nbsp;     | &nbsp; |
+| ----------- | ----------- |
+| **Partner Name**      | SIB Visions     |
+| **Partner Product**   |  VisionX       |
+| **Web Site**      |    https://visionx.sibvisions.com/    |
+| **Version & Platform**   |  VisionX 5.6.925 : CentOS7 / Windows 10      |
+| **Product Description**      |  VisionX is a low-code platform for creating web and mobile applications quickly and easily. VisionX and its Oracle Forms Migration Extension are an efficient solution for semi-automated migration of Oracle Forms and Oracle APEX into modern web and mobile applications. Together with an Oracle database to EDB Postgres Advanced Server migration, VisionX provides a fast, smooth, and reliable Oracle exit strategy.   |
```

Comment:
Change 
Together with an Oracle database to EDB Postgres Advanced Server migration, VisionX provides a fast, smooth, and reliable Oracle exit strategy.
To
VisionX, together with  EDB Postgres Advanced Server migration, can help provide an exit strategy from Oracle to Postgres.